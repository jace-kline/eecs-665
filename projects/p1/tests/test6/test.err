FATAL [1,6]: Illegal character \
