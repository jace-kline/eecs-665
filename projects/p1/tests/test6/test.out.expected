INTLIT:123 [1,13]
STRINGLIT:"Let's put a space character in a char literal" [1,17]
CHARLIT: 
EOF [1,67]